### DISCUSSION TITLE
Welcome to salesboard-app Discussions!

### DISCUSSION THREAD

**Discussion by paulaFenner** [paulaFenner/salesboard-app#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by abaltazapinto** [comment]
just strating the usual HTML last commit on the html markup 
![image](https://github.com/user-attachments/assets/c9424991-101f-4fa2-9bb4-f4c3fbd8c69e)
It will be good idea if we can set up a user profile within the sales board app , with main image for each one !

**Reply by paulaFenner** [reply]
That could be a nice stretch goal for us to work. 👏
Create 3 separate users and account individual sales.
